scripts/exregional_run_anal.sh

```diff
@@ -0,0 +1,968 @@
+#
+# decide regional_ensemble_option: global ensemble (1) or FV3LAM ensemble (5)
+#
+#---------------------------------------------------------------------
+#
+echo "regional_ensemble_option is ",${regional_ensemble_option:-1}
+
+print_info_msg "$VERBOSE" "FIXgsi is $FIXgsi"
+print_info_msg "$VERBOSE" "pregen_grid_dir is $pregen_grid_dir"
+print_info_msg "$VERBOSE" "default bkpath is $bkpath"
+print_info_msg "$VERBOSE" "background type is is $BKTYPE"
+
+#
+# Check if we have enough FV3-LAM ensembles when regional_ensemble_option=5
+#
+if  [[ ${regional_ensemble_option:-1} -eq 5 ]]; then
+  ens_nstarthr=$( printf "%02d" ${DA_CYCLE_INTERV} )
+  imem=1
+  ifound=0
+  for hrs in ${CYCL_HRS_HYB_FV3LAM_ENS[@]}; do
+  if [ $HH == ${hrs} ]; then
+
+  while [[ $imem -le ${NUM_ENS_MEMBERS} ]];do
+    memcharv0=$( printf "%03d" $imem )
+    memchar=mem$( printf "%04d" $imem )
+
+    YYYYMMDDHHmInterv=$( date +%Y%m%d%H -d "${START_DATE} ${DA_CYCLE_INTERV} hours ago" )
+    restart_prefix="${YYYYMMDD}.${HH}0000."
+    SLASH_ENSMEM_SUBDIR=$memchar
+    bkpathmem=${RRFSE_NWGES_BASEDIR}/${YYYYMMDDHHmInterv}/${SLASH_ENSMEM_SUBDIR}/fcst_fv3lam/RESTART
+    if [ ${DO_SPINUP} == "TRUE" ]; then
+      for cycl_hrs in ${CYCL_HRS_PRODSTART_ENS[@]}; do
+       if [ $HH == ${cycl_hrs} ]; then
+         bkpathmem=${RRFSE_NWGES_BASEDIR}/${YYYYMMDDHHmInterv}/${SLASH_ENSMEM_SUBDIR}/fcst_fv3lam_spinup/RESTART
+       fi
+      done
+    fi
+    dynvarfile=${bkpathmem}/${restart_prefix}fv_core.res.tile1.nc
+    tracerfile=${bkpathmem}/${restart_prefix}fv_tracer.res.tile1.nc
+    phyvarfile=${bkpathmem}/${restart_prefix}phy_data.nc
+    if [ -r "${dynvarfile}" ] && [ -r "${tracerfile}" ] && [ -r "${phyvarfile}" ] ; then
+      ln_vrfy -snf ${bkpathmem}/${restart_prefix}fv_core.res.tile1.nc       fv3SAR${ens_nstarthr}_ens_mem${memcharv0}-fv3_dynvars
+      ln_vrfy -snf ${bkpathmem}/${restart_prefix}fv_tracer.res.tile1.nc     fv3SAR${ens_nstarthr}_ens_mem${memcharv0}-fv3_tracer
+      ln_vrfy -snf ${bkpathmem}/${restart_prefix}phy_data.nc                fv3SAR${ens_nstarthr}_ens_mem${memcharv0}-fv3_phyvars
+      (( ifound += 1 ))
+    else
+      print_info_msg "Error: cannot find ensemble files: ${dynvarfile} ${tracerfile} ${phyvarfile} "
+    fi
+    (( imem += 1 ))
+  done
+ 
+  fi
+  done
+
+  if [[ $ifound -ne ${NUM_ENS_MEMBERS} ]] || [[ ${BKTYPE} -eq 1 ]]; then
+    print_info_msg "Not enough FV3_LAM ensembles, will fall to GDAS"
+    regional_ensemble_option=1
+  fi
+fi
+#
+if  [[ ${regional_ensemble_option:-1} -eq 1 ]]; then #using GDAS
+  #-----------------------------------------------------------------------
+  # Make a list of the latest GFS EnKF ensemble
+  #-----------------------------------------------------------------------
+  stampcycle=$(date -d "${START_DATE}" +%s)
+  minHourDiff=100
+  loops="009"    # or 009s for GFSv15
+  ftype="nc"  # or nemsio for GFSv15
+  foundgdasens="false"
+  #cat "no ens found" >> filelist03
```

Comment:
@MichaelLueken RRFS_dev1 has that line activated, but it seems to be a bug since you can't cat a string. 
https://github.com/NOAA-GSL/regional_workflow/blob/feature/RRFS_dev1/scripts/exregional_run_analysis.sh#L266
SRW being equipped with debugging mode with `set -eux`, it was able to capture the problem. I wanted to report the issue in GSL repository, but they don't seem to have issues tab activated.